[5.3.0] undefined symbol: php_hash_bin2hex

Build phpredis 5.3.0, ./configure --enable-redis --disable-redis-session

"**make**" got **WARNING**:
> /root/sources/redis-5.3.0/redis.c: In function ‘redis_random_hex_bytes’:
> /root/sources/redis-5.3.0/redis.c:787:9: warning: implicit declaration of function ‘php_hash_bin2hex’; did you mean ‘zend_hash_find_ex’? [-Wimplicit-function-declaration]
>          php_hash_bin2hex(dst, (unsigned char *)ZSTR_VAL(s), bytes);
>          ^~~~~~~~~~~~~~~~
>          zend_hash_find_ex

"**make test**" got "**undefined symbol**":
> /myserver/php-7.3.19/bin/php: symbol lookup error: /root/sources/redis-5.3.0/modules/redis.so: undefined symbol: php_hash_bin2hex
> /myserver/php-7.3.19/bin/phpdbg: symbol lookup error: /root/sources/redis-5.3.0/modules/redis.so: undefined symbol: php_hash_bin2hex
> /myserver/php-7.3.19/bin/php: symbol lookup error: /root/sources/redis-5.3.0/modules/redis.so: undefined symbol: php_hash_bin2hex

### I'm seeing this behaviour on
- OS: CentOS 8
- PHP: 7.3.19
- phpredis: 5.3.0
- gcc version 8.3.1 20191121 (Red Hat 8.3.1-5) (GCC)

Actually - I had both session and hash - but the build threw the following warning:

```
--- redis.lo ---
/var/ports/basejail/usr/ports/databases/pecl-redis/work-php72/redis-5.3.0/redis.c:787:9: warning: implicit declaration of function 'php_hash_bin2hex' is invalid in C99 [-Wimplicit-function-declaration]
        php_hash_bin2hex(dst, (unsigned char *)ZSTR_VAL(s), bytes);
        ^
--- redis_sentinel.lo ---
```

Adding the include to redis.c - fix this, and it seems to work.

--- a/redis.c
+++ b/redis.c
@@ -30,6 +30,7 @@
 #include <standard/php_random.h>
 #include <zend_exceptions.h>
 #include <ext/standard/info.h>
+#include <ext/hash/php_hash.h>
 
 
 #ifdef PHP_SESSION

